iv.yaml

```diff
@@ -0,0 +1,9 @@
+lenguaje: node.js
+objeto valor: 
+  - src/user.js
+  - src/film.js
+  - src/user_collection.js
```

Comment:
... pero ya que lo habéis puesto, una colección es imposible que sea un objeto valor porque no es inmutable.